publisher.go

```diff
@@ -122,6 +123,8 @@ func buildInsertQuery(queue string, msgCount int) string {
 		sb.WriteString(params.Next())
 		sb.WriteString(",")
 		sb.WriteString(params.Next())
+		sb.WriteString(",")
+		sb.WriteString(fmt.Sprintf("NOW()+make_interval(secs => %s)", params.Next()))
 		sb.WriteString(")")
```

Comment:
I would avoid the `fmt` call
```suggestion
		sb.WriteString(",NOW()+make_interval(secs => ")
		sb.WriteString(params.Next())
		sb.WriteString("))")
```

I thought of making it conditional, but we plan to add prepared statements to speed up the inserts, so hard-coded may be better :) WDYT?

Reply:
Agree - using fmt everywhere is kinda my habbit, but there is no practical benefit in this case.